src/components/search-form/search-form.js

```diff
@@ -106,11 +112,11 @@ class SearchForm extends Component {
     this.props.onFilterChange(sort, searchFilter);
   };
 
-  handleUpdateTagFilter = (filter) => {
-    const tagFilter = {
-      tags: filter.values.length > 0 ? filter.values.join(',') : undefined
+  handleStandaloneFilterChange = filter => {
+    const formattedFilter = {
+      [filter.name]: filter.values.length > 0 ? filter.values.join(',') : undefined
```

Comment:
I think here could be used chaining operator
```suggestion
      [filter.name]: filter?.values.join(',') || undefined
```
or without `undefined` if empty string is possible

Reply:
we need explicit `undefined` here, but to write `filter.values.join(',') || undefined` is a nice suggestion